src/aspire/utils/misc.py

```diff
@@ -48,6 +50,31 @@ def importlib_path(package, resource):
     return p
 
 
+def timestamp_filename(filepath, move=True):
```

Comment:
Not obvious from the name of the function that this actually interacts with the filesystem (by default). Maybe something like `rename_with_timestamp`?

Reply:
Sure. Done!